src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1,4 +1,34 @@
+import classNames from 'classnames';
+import { UserInfo } from '../UserInfo/UserInfo';
+
+import users from '../../api/users';
+
+import { Todo } from '../../types/Todo';
+import { User } from '../../types/User';
+
 // Add the required types and props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
-);
+
+function findPeople(user: User | null, userId: number) {
+  return users
+    .find((people) => people.id === userId || people.name === user?.name);
```

Comment:
It's better to specify what will be if the find method will not find nothing 
```suggestion
    .find((people) => people.id === userId || people.name === user?.name) || null;
```

Reply:
Added. Thanks